.gitignore

```diff
@@ -182,3 +182,6 @@ node_modules/
 
 # Mac Desktop Services Store
 *.DS_Store
+
+# Landing zone data storage
+# /data/landing_zone/*
```

Comment:
I'm not sure whether we're intending to keep `data/landing_zone/` under version control given that this is commented out. Do we want to keep the directory and the README.md under VC and ignore everything else in the folder? If so I think we can accomplish that as such in the .gitignore file:

`data/landing_zone/*`
`!data/landing_zone/README.md`